### PR TITLE
Improve plan documentation

### PR DESCRIPTION
**Pull request by lulf**
* Add image describing relationship between different service
configuration entities.
* Add introductory paragraph to the 'Configuring' section of the service
admin book.
* Improve plan examples and inline.

### CODE REVIEW

File: documentation/assemblies/assembly-configuring.adoc [hunk 1]
```diff
@@ -4,6 +4,20 @@
 [id='configuring-{context}']
 = Configuring {ProductName}
 
+The service operator configures {ProductName} by defining resources constituting the "service configuration". This configuration contains instances of the following resource types:
+
+* `AuthenticationService`  - Describes an authentication service instance used to authenticate messaging clients.
+* `AddressSpacePlan` - Describes the messaging resources available for address spaces using this plan, such as the available address plans, the amount of router and broker resources that can be used.
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-configuring.adoc]:
```suggestion
* `AddressSpacePlan` - Describes the messaging resources available for address spaces using this plan, such as the available address plans and the amount of router and broker resources that can be used.
```

File: documentation/assemblies/assembly-configuring.adoc [hunk 2]
```diff
@@ -4,6 +4,20 @@
...
+* `AddressPlan` - Describes the messaging resources consumed by a particular address using this plan, such as what fraction of routers and brokers an address will use and other properties that should be set for multiple addresses.
+* `StandardInfraConfig` - Describes the router and broker configuration for the `standard` address space type such as memory limits, storage capacity, affinity and more.
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-configuring.adoc]:
```suggestion
* `StandardInfraConfig` - Describes the router and broker configuration for the `standard` address space type such as memory limits, storage capacity, affinity, and more.
```

File: documentation/assemblies/assembly-configuring.adoc [hunk 3]
```diff
@@ -4,6 +4,20 @@
...
+* `BrokeredInfraConfig` - Describes the broker configuration for the `brokered` address space type such as memory limits, storage capacity, affainity and more.
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-configuring.adoc]:
```suggestion
* `BrokeredInfraConfig` - Describes the broker configuration for the `brokered` address space type such as memory limits, storage capacity, affinity, and more.
```

File: documentation/assemblies/assembly-configuring.adoc [hunk 4]
```diff
@@ -4,6 +4,20 @@
...
+
+When created, these resources defines the configuration that are available to the messaging tenants.
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-configuring.adoc]:
```suggestion
When created, these resources define the configuration that is available to the messaging tenants.
```

File: documentation/modules/con-address-space-plans.adoc [hunk 5]
```diff
@@ -5,24 +5,39 @@
...
-
-* `metadata.name`
-* `spec.resourceLimits`
-* `spec.addressPlans`
-* `spec.addressSpaceType`
-* `spec.infraConfigRef`
+<1> A reference to the `StandardInfraConfig` (for the `standard` address space type) or the `BrokeredInfraConfig` (for the `brokered` address space type) describing the infrastructure deployed for address spaces using this plan.
+<2> The address space type this plan applies to, either `standard` or `brokered`.
+<3> A list of address plans available to address spaces using this plan.
+<4> The max amount of `router` and `broker` for address spaces using this plan. For the `brokered` address space type, only the `broker` field is required.
```
Comment (jenmalloy) [inline documentation/modules/con-address-space-plans.adoc]:
```suggestion
<4> The maximum number of routers (`router`) and brokers (`broker`) for address spaces using this plan. For the `brokered` address space type, only the `broker` field is required.
```

**Comment by lulf** [comment]
@jenmalloy , added changes

**Pull request merged**